src/svg/arrow-left.svg

```diff
@@ -1 +1 @@
-<svg focusable="false" viewBox="0 0 1000 1000"><path d="M336.2 274.5l-210.1 210h805.4c13 0 23 10 23 23s-10 23-23 23H126.1l210.1 210.1c11 11 11 21 0 32-5 5-10 7-16 7s-11-2-16-7l-249.1-249c-11-11-11-21 0-32l249.1-249.1c21-21.1 53 10.9 32 32z"/></svg>
+<svg focusable="false" aria-hidden="true" viewBox="0 0 1000 1000"><path d="M336.2 274.5l-210.1 210h805.4c13 0 23 10 23 23s-10 23-23 23H126.1l210.1 210.1c11 11 11 21 0 32-5 5-10 7-16 7s-11-2-16-7l-249.1-249c-11-11-11-21 0-32l249.1-249.1c21-21.1 53 10.9 32 32z"/></svg>

```

Comment:
looks like our build is breaking on this; specifically the `buildCSS.js` script.

Reply:
is there anything I can help with here?

Reply:
I'm not sure :-/ it's likely that we'll need to make another fix before this can be merged; if you wanted to look into the failure and help figure that out, that'd be great!

Reply:
Hey @ljharb
Who might help me with this PR's build failure?